main.go

```diff
@@ -84,6 +82,7 @@ func main() {
 	flag.Parse()
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+	setupLog.Info("Initializing", "gitVersion", version.GitVersion, "gitCommit", version.GitCommit)
```

Comment:
`setupLog` implements some promise logic. The promise is supposed to log once the logger is ready.

But if it doesn't work, we don't need to debug controller-runtime's log.

/lgtm
/approve